crates/block-producer/src/server.rs

```diff
@@ -361,11 +362,11 @@ mod test {
             BlockProducer {
                 block_producer_address: block_producer_addr,
                 store_address: store_addr,
-                ntx_builder_address,
                 batch_prover_url: None,
                 block_prover_url: None,
                 batch_interval: Duration::from_millis(500),
                 block_interval: Duration::from_millis(500),
+                ntx_builder_address,
```

Comment:
nit: should we use `_addr` suffix (sims like that's what we use in other cases)? Or maybe we should just use `_address` suffix eveywhere?

Also, I'd probably move this up to be after `store_address`.